app/controllers/db/works_controller.rb

```diff
@@ -0,0 +1,82 @@
+class Db::WorksController < Db::ApplicationController
+  permits :season_id, :sc_tid, :title, :media, :official_site_url, :wikipedia_url,
+          :twitter_username, :twitter_hashtag, :released_at, :released_at_about,
+          :fetch_syobocal
+
+  before_action :authenticate_user!, only: [:new, :create, :edit, :update, :destroy]
+
+  def index(page: nil)
+    @works = Work.order(id: :desc).page(page)
+  end
+
+  def season(page: nil, slug: ENV["ANNICT_CURRENT_SEASON"])
+    @works = Work.by_season(slug).order(id: :desc).page(page)
+    render :index
+  end
+
+  def resourceless(page: nil, name: "episode")
+    @works = case name
+             when "episode" then Work.where(episodes_count: 0)
+             when "item" then Work.where(items_count: 0)
+             end
+    @works = @works.order(watchers_count: :desc).page(page)
+    render :index
+  end
+
+  def search(page: nil, q: "")
+    @works = Work.where("lower(title) LIKE ?", "%#{q}%")
+      .order(id: :desc)
+      .page(page)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.